src/main/java/uk/gov/di/config/RPConfig.java

```diff
@@ -9,12 +9,25 @@ public record RPConfig(
         String idTokenSigningAlgorithm,
         String serviceName,
         String opBaseUrl,
-        String tokenClientSecret) {
+        String tokenClientSecret,
+        String inheritedIdentityJwtSigningKey) {
     public String authCallbackUrl() {
         return Configuration.getStubUrl() + "/oidc/authorization-code/callback";
     }
 
     public String postLogoutRedirectUrl() {
         return Configuration.getStubUrl() + "/signed-out";
     }
+
+    public static String getInheritedIdentityJwtIssuer() {
+        return "https://orch.stubs.account.gov.uk/migration/v1";
+    }
+
+    public static String getInheritedIdentityJwtVtm() {
+        return "https://hmrc.gov.uk/trustmark";
+    }
+
+    public static String getIpvEndpoint() {
+        return "https://identity.build.account.gov.uk/";
+    }
```

Comment:
I reckon these would make more sense on `Configuration`, as they're not RP specific. What do you think? 

I'll expand the config model at some point to have generic config as well as RP specific stuff